varianteval/scripts/annotation_script.py

```diff
@@ -0,0 +1,124 @@
+"""
+Script to add annotation to input callset with respect to specified
+comparison callsets
+"""
+
+import sys
+import re
+import numpy as np
+from pysam import VariantFile
+import argparse
+from collections import defaultdict
+
+
+def overlapping_records(intrvl_A, intrvl_B, overlap_threshold=0.5, bp_margin=5000):
```

Comment:
nit: rename to "interval" overlap rather than "record" -- this method is not aware about records since the inputs are intervals